src/EditorFeatures/Core/Implementation/Diagnostics/DiagnosticsClassificationTaggerProvider.cs

```diff
@@ -17,8 +17,7 @@
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Diagnostics
 {
     [Export(typeof(ITaggerProvider))]
-    [ContentType(ContentTypeNames.CSharpContentType)]
-    [ContentType(ContentTypeNames.VisualBasicContentType)]
+    [ContentType(ContentTypeNames.RoslynContentType)]
```

Comment:
@Pilchie had suggested in one of @CyrusNajmabadi 's PRs to convert more generally useful exports to RoslynContentType instead of adding individual languages. This one seems useful to all languages.